packages/handler-kit-azure-func/src/logger.ts

```diff
@@ -0,0 +1,23 @@
+import * as azure from "@azure/functions";
+
+import * as L from "@pagopa/logger";
+
+// Derive a concrete implementation L.Logger using azure.Context.log
+export const getLogger = (ctx: azure.Context): L.Logger => ({
+  log: (s, level) => () => {
+    const logFunc: Record<typeof level, (s: string) => void> = {
+      debug: ctx.log.verbose,
+      info: ctx.log.info,
+      warn: ctx.log.warn,
+      error: ctx.log.error,
+      fatal: ctx.log.error,
+    };
+    logFunc[level](s);
+  },
+  format:
+    process.env.NODE_ENV === "development" ? L.format.simple : L.format.json,
```

Comment:
if `process.env.NODE_ENV` is not set, is it correct not to have the simple format?

Reply:
yess! the default should be always the structured log